### PR TITLE
Celery running phase-to-phase migrations

### LINKED ISSUE #1756: Add scheduling for phase-to-phase migrations via celery
Add scheduling for phase-to-phase migrations via celery

### PR DESCRIPTION
**Pull request by ckcollab**
resolves #1756

### CODE REVIEW

**Comment by ckcollab** [comment]
@Tthomas63 todo list:

- [ ] Review my changes
- [ ] Enable the actual phase migration behavior, checking it against `create_competition_zip`
- [ ] Set the migrations to run every 5 minutes
- [ ] Assign "Reviewers" back to me when you're ready for me to review your stuff
- [ ] Enjoy your weekend! :)

**Comment by Tthomas63** [comment]
@ckcollab 

I cannot set you back to the reviewer, but it is ready.

File: codalab/apps/web/tasks.py [hunk 1]
```diff
@@ -769,3 +768,13 @@ def send_mass_email(competition_pk, body=None, subject=None, from_email=None, to
...
+
+
+@task(queue='site-worker')
+def do_phase_migrations():
+    logger.info("Task: Do phase migrations running.")
+    competitions = Competition.objects.filter(is_migrating=False)
+
+    for c in competitions:
+        logger.info("Checking future phase submissions for " + str(c))
+        c.check_future_phase_sumbmissions()
```
Comment (ckcollab) [inline codalab/apps/web/tasks.py]: Missing newline at end of file

Comment (ckcollab) [inline codalab/apps/web/tasks.py]: (I fixed it :))

**Pull request merged**